App.js

```diff
@@ -43,10 +43,8 @@ const App = () => {
               <Text style={styles.sectionTitle}>Step One</Text>
               <Text style={styles.sectionDescription}>
                 Edit <Text style={styles.highlight}>App.js</Text> to change this
```

Comment:
Should make a pull from the current `master` branch for the navigation (`<MainTabScreen />`)

Reply:
> Should make a pull from the current `master` branch for the navigation (`<MainTabScreen />`)

done : no issues